kubernetes/istio-install/istio-mixer.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
```

Comment:
Manager actually doesn't require all pods to belong to services. Auth and mixer might require this.